src/server/views/dashboard/users-edit.njk

```diff
@@ -52,6 +52,28 @@
     </div>
     <button class="govuk-button" data-module="govuk-button">Save</button>
   </form>
-
+  {% if user.lists.length > 0 %}
+    <div class="govuk-body">
+      This user is an admin on <strong>{{user.lists.length}}</strong> {{"lists" if user.lists.length > 1 else "list"}}
+    </div>
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Type</th>
+          <th scope="col" class="govuk-table__header govuk-table__header--numeric">Country</th>
```

Comment:
Looks strange imo - can we do a left align instead? Right align is usually more helpful for numeric display

Reply:
Should I double check with Georgia?